spec.md

```diff
@@ -2273,6 +2424,10 @@ message NodeServiceCapability {
       GET_VOLUME_STATS = 2;
       // See VolumeExpansion for details.
       EXPAND_VOLUME = 3;
+      // If Plugin implements VOLUME_HEALTH capability
```

Comment:
ditto: should probably appear in AlphaFeatures instead

Reply:
Done